routing/pathfind.go

```diff
@@ -353,22 +354,25 @@ func getMaxOutgoingAmt(node route.Vertex, outgoingChan *uint64,
 		// fail the pathfinding early.
 		if !ok {
 			max = lnwire.MaxMilliSatoshi
+			total = lnwire.MaxMilliSatoshi
 			return nil
 		}
 
 		if bandwidth > max {
 			max = bandwidth
 		}
 
+		total += bandwidth
```

Comment:
So we'll get an under flow here if the target channel isn't one of ours?

Reply:
Or did you intend to only increment the total here by the current max payment size?

Reply:
How do you mean underflow? In the code - at least before the fix above - it would return immediately after encountering a channel without bandwidth hints. 

This function is only called for the self node and `bandwidthHints` only contains local channel balances.

Reply:
Resolved now, but I mean the implications of adding `math.MaxUint32` to an existing integer.